metadata-ingestion/src/datahub/ingestion/source/bigquery_usage.py

```diff
@@ -284,7 +284,7 @@ def _parse_bigquery_log_entries(
         self, entries: Iterable[AuditLogEntry]
     ) -> Iterable[Union[ReadEvent, QueryEvent]]:
         for entry in entries:
-            event: Union[ReadEvent, QueryEvent]
+            event: Union[None, ReadEvent, QueryEvent] = None
             if ReadEvent.can_parse_entry(entry):
```

Comment:
There are other log types in here aside from Read and Query?

Reply:
Nope, since the filter we use on the logs should only return these two types

https://github.com/linkedin/datahub/blob/835c0110020354d32b925b8c2221cc3e60ab3017/metadata-ingestion/src/datahub/ingestion/source/bigquery_usage.py#L42-L64